editor/editor.cpp

```diff
@@ -209,6 +204,8 @@ void Editor::drawDefaultUI()
 					ImGui::SameLine();
 					if (ImGui::Button("Create"))
 					{
+						if (ProjectManager::GetSingleton()->isAnyLevelOpen())
+							saveAll(nullptr);
```

Comment:
Use this format for all `if` statements. Don't remove curly braces, it helps reduce bugs in the future:
```cpp
if (condition)
{
    // do something
}
```